### PR TITLE
Employee attendance change request

### PR DESCRIPTION
**Pull request by varinderdhanda**
I have added a function to raise a request by the employee to change the attendance such as replace Absent with a leave. Give me your reviews.

### CODE REVIEW

**Review by jagdeepbaidwan — COMMENTED** [review]
In  employeee_management.c line 23 in attendance_change function is it possible to add some validation to date being given by user ?

File: src/employee_attdnce_management.c [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+	char stmt[1500];
+	//struct request
+	//{
+		int id;
+       	char date[15];
+       	char description[150];
+	//};
+	//struct request req;
+	printf("Enter the date on which you have problem (mm/dd/yyyy)\n");
+    scanf("%s",date);
```
Comment (Arshpreet-Singh38) [inline src/employee_attdnce_management.c:23]: Date validation can be provided here.

File: src/employee_attdnce_management.c [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+   		return "Failed to execute query.";
+   	}
+    else
+ 	{
+ 	   return "\n\n\n Record entered in database\n\n\n\n";
+ 	}
+}
+//Function attendance request change
+
+void reuest_status(int emp_id)
```
Comment (Arshpreet-Singh38) [inline src/employee_attdnce_management.c:45]: Typo error for "request_status"

File: src/employee_attdnce_management.c [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+		printf("                Press 3 Request attendance change\n");
+		int ch;
+		scanf("%d",&ch);
+		if(ch==1)
+		{
+
+		}
+		else if(ch==2)
+		{
+			reuest_status(emp_id);
```
Comment (Arshpreet-Singh38) [inline src/employee_attdnce_management.c:127]: Type error "request_status"

**Pull request merged**